docs/source/admin/capacity-planning.md

```diff
@@ -0,0 +1,246 @@
+# Capacity planning
+
+General capacity planning advice for JupyterHub is hard to give,
```

Comment:
Point taken! I generally follow "semantic line breaks" for exactly that reason, but break on clauses rather than only full sentences. I can do sentences, though!

Reply:
I am also a fan of semantic breaks :-) I've just found that for some reason, it is more common for me to want to edit at the sentence-level rather than the clause level, so diffing one line corresponding to a sentence is the compromise I use. Either one is fine, I just wanted to note it

Reply:
Yeah, I mostly thought about reading/editing a fixed diff, but sentences makes total sense in the context of tools like GitHub suggestions. I hadn't thought of that.